Core/include/Acts/EventData/ParameterConcept.hpp

```diff
@@ -64,16 +64,19 @@ namespace concept {
         identical_to<double, time_returntype_t, const P>;
     constexpr static bool surfacebound_covariance_exists =
         identical_to<const std::optional<BoundSymMatrix>&,
-                     covariance_returntype_t, const P>;
+                     covariance_returntype_t,
+                     const P>;
     constexpr static bool surfacebound_parameters_exists =
         identical_to<BoundVector, parameters_returntype_t, const P>;
 
     /// Tie all surface bound results together
-    constexpr static bool boundValue =
-        require<reference_surface_exists, surfacebound_position_exists,
-                surfacebound_momentum_exists, surfacebound_charge_exists,
-                surfacebound_time_exists, surfacebound_covariance_exists,
-                surfacebound_parameters_exists>;
+    constexpr static bool boundValue = require<reference_surface_exists,
+                                               surfacebound_position_exists,
+                                               surfacebound_momentum_exists,
+                                               surfacebound_charge_exists,
+                                               surfacebound_time_exists,
+                                               surfacebound_covariance_exists,
+                                               surfacebound_parameters_exists>;
```

Comment:
We used to have this, right? I'm personally not a huge caon of this.

Reply:
I'm surprised you took an example where it's so obviously much more readable :D
If people really disagree with how this block looks before/after then there's nothing I can add to that, so I'm happy to use this as advertisement. Please cast your vote as reaction to the PR description with a thumbs up or down. voting closes next tuesday at 16:59 :)

Reply:
If you'd like to change something else or modify the changed bits, please write them as a comment and we'll cast our vote on that until next tuesday as well.